tiledb/sm/storage_manager/context_resources.h

```diff
@@ -111,6 +111,10 @@ class ContextResources {
     return rest_client_;
   }
 
+  [[nodiscard]] inline MemoryTrackerManager& memory_tracker_manager() const {
```

Comment:
I added a getter for the memory tracker manager, instead of adding the same budget and callback parameters to `create_memory_tracker`.